docs/source/pages/howToWorkWithPlatform/mainApplication/templateList/templateManagement/documentManagementOnTemplate/fields/signature/signature.rst

```diff
@@ -34,33 +35,41 @@ When all attributes are set, you can click Save button and field will be added.
 How to fill a signature field in the envelope
 =============================================
 
-1. Open envelope where signature field is located
+1. Open envelope where signature field is located.
 2. Click on signature field
 
 .. image:: pic_signature/signatureEnvelope.png
    :width: 600
    :align: center
 
-3. Window will appear
+3. Window will appear.
 
 .. image:: pic_signature/signatureDraw.png
    :width: 600
    :align: center
 
 .. note:: If you have saved e-ink signature in your profile settings page, window will not open and signature will be automatically applied.
 
-4. Draw your signature on the empty area by pressing left side on the mouse and dragging it around. You have to draw at least 0.5 sec to complete validation of the field. Note that you can change color or input text instead of drawing on Text tab
+4. Draw your signature on the empty area by pressing left side on the mouse and dragging it around. You have to draw at least 0.5 sec to complete validation of the field. Note that you can change color or input text instead of drawing on Text tab.
 
 .. image:: pic_signature/signatureDrawn.png
    :width: 600
    :align: center
 
-5. You can reset drawing area with "Clear" button
-6. Click "Sign" to insert your signature to the field (can't be empty)
-7. After clicking "Save" you will see your signature in the document
+5. You can reset drawing area with "Clear" button.
+6. Click "Sign" to insert your signature to the field (can't be empty) and you will see your signature in the document.
+7. Under the signature, you will see its details, depending on whether the "Show signature details" toggle was enabled or not on the template. 
 
-.. image:: pic_signature/documentSigned.png
+.. note:: If toggle was enabled - there will be displayed signature hash and 'Signed by [instance name]' and reserved space for date and mailbox or email address of the signer (it will appear on the document after sending the envelope). If toggle was disabled there will be just signature hash and 'Signed by [instance name]'. 
```

Comment:
"reserved space for date and mailbox or email address" -> reserved space for date and email address

Reply:
done